main.go

```diff
@@ -287,23 +293,39 @@ func checkBuf(path string, src []byte) ([]byte, error) {
 // exception is made for cgo, whose "C" psuedo-imports are extracted into
 // separate import declarations.
 func remapImports(file *parser.File) map[*parser.ImportDecl][]render.ImportBlock {
-	var (
-		stdlibImports []parser.ImportSpec
-		otherImports  []parser.ImportSpec
-	)
-
-	for _, imp := range file.ImportSpecs() {
-		switch impPath := imp.Path(); {
-		case impPath == "C":
-			continue
-		case strings.Contains(impPath, "."):
+	imports := file.ImportSpecs()
+	stdlibImports := make([]parser.ImportSpec, 0, len(imports))
+	otherImports := make([]parser.ImportSpec, 0, len(imports))
+	localImports := make([]parser.ImportSpec, 0, len(imports))
+
+NEXT_IMPORT:
+	for _, imp := range imports {
+		impPath := imp.Path()
+		if impPath == "C" {
+			continue NEXT_IMPORT
+		}
+
+		if localPrefix != nil && *localPrefix != "" {
+			for _, lp := range strings.Split(*localPrefix, ",") {
```

Comment:
Can you move this split to happen only once.  
Then, you can just range over that list and it you won't need to do the double if beforehand.